homework3/index.js

```diff
@@ -0,0 +1,66 @@
+//---------------task 1----------------
+function durationBetweenDates(firstDate, secondDate, dimension) {
+    const date1 = new Date(firstDate);
+    const date2 = new Date(secondDate);
+    const diffTime = Math.abs(date1 - date2);
+    if (dimension === 'seconds') {
+        const diffSeconds = Math.ceil(diffTime / (1000));
+        return console.log(diffSeconds);
+    }
+    if (dimension === 'minutes') { 
+        const diffMinutes = Math.ceil(diffTime / (1000 * 60 * 60));
+        return console.log(diffMinutes);
+    }
+    if (dimension === 'hours') {
+        const diffHours = Math.ceil(diffTime / (1000 * 60));
+        return console.log(diffHours);
+    }
+    if (dimension === 'days') {
+        const diffDays = Math.ceil(diffTime / (1000 * 60 * 60 *24));
+        return console.log(diffDays);
+    }
```

Comment:
This is a working solution. This is good. Let's begin refactoring :)
Do you see a lot of copy/past here?
Let's try to create a map object with dimensions. Maybe this can help you to avoid this copy/pasts.